tests/e2e/user-journeys.spec.ts

```diff
@@ -10,6 +10,26 @@ test.describe('User journeys', () => {
   test('serves a response from the edge when using @netlify/remix-edge-adapter', async ({ page, edgeSite }) => {
     const response = await page.goto(edgeSite.url)
     await expect(page.getByRole('heading', { name: /Welcome to Remix/i })).toBeVisible()
-    expect(response?.headers()['x-nf-edge-functions']).toBe('server')
+    expect(response?.headers()['x-nf-edge-functions']).toBe('remix-server')
+  })
+
+  test.describe('classic Remix compiler', () => {
```

Comment:
nit - We have classic `Remix compiler` at the same test level as `User journeys`, perhaps we could name them to be both about compilers?

Reply:
My mistake!